tests/integration/Mvc/Model/Behavior/NotifyCest.php

```diff
@@ -1,33 +0,0 @@
-<?php
```

Comment:
Why are these deleted?

Reply:
It's an abstract class so we can't test it directly and it is covered by the subclasses SoftDelete and Timestampable anyway.